framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -348,6 +348,11 @@
     // -Acfgviz=MyClass,verbose
     "verbosecfg",
 
+    /// LSP Support
+
+    // Whether to print refined types for local variables
+    "printTypeRefinements",
```

Comment:
How about `outputAnnotatedTypes` to reflect that this isn't just for type refinements?
Or `outputImplicitAnnotations`, to indicate it's not just type annotations? ("Defaulted" would be too limiting, but "Implicit" is maybe also not the best term. Maybe "outputEffectiveAnnotations"?)